qemu/tests/cluster_size_check.py

```diff
@@ -53,22 +53,20 @@ def check_cluster_size(parttern, expect, csize_set):
 
         if status_error:
             if result.exit_status == 0:
-                logging.error("Return success with invalid Cluster size: %s"
+                logging.error("Create image sucessfully with invalid size: %s"
                               % csize_set)
-                logging.error("%s.\n" % result)
                 cfail += 1
+                fail_log += "Succeed in creating image unexpectedly.\n"
         else:
             output = image.info()
             error.context("Check the cluster size from output", logging.info)
             cluster_size = re.findall(parttern, output)
             if cluster_size:
                 if cluster_size[0] != expect:
-                    logging.error("Cluster size mismatch")
-                    logging.error("Cluster size report by command: %s"
-                                  % cluster_size)
-                    logging.error("Cluster size expect: %s" % expect)
+                    logging.error("Cluster size %s is not expected value %s"
```

Comment:
Thanks for your comment.
IMHO, 
The definition of mismatch is something that was not correctly **paired.**
and we compare cluster_size with *expect*.
so I think " is not expected value" more better.